File: src/Services/API/Domain/Functions/SqlFunctionParameter.cs
namespace API.Domain.Functions
{
    public class SqlFunctionParameter
    {
        public string Parameter_name { get; set; }
        public string Type { get; set; }
        public int Length { get; set; }
        public int Prec { get; set; }
        public int Scale { get; set; }
        public int Param_order { get; set; }
        public string Collation { get; set; }
        public string ExtendedProperty { get; set; }
    }

}

File: src/WebClient/mssql.client/src/app/function/model/SqlFunctionParameter.ts
/**
 * Represents a parameter for a SQL function.
 */
export class SqlFunctionParameter {
  Parameter_name: string;
  Type: string;
  Length: number;
  Prec: number;
  Scale: number;
  Param_order: number;
  Collation: string;
  ExtendedProperty: string;

  constructor(
    Parameter_name: string,
    Type: string,
    Length: number,
    Prec: number,
    Scale: number,
    Param_order: number,
    Collation: string,
    ExtendedProperty: string
  ) {
    this.Parameter_name = Parameter_name;
    this.Type = Type;
    this.Length = Length;
    this.Prec = Prec;
    this.Scale = Scale;
    this.Param_order = Param_order;
    this.Collation = Collation;
    this.ExtendedProperty = ExtendedProperty;
  }
}
